controllers/device_controller.go

```diff
@@ -232,16 +215,16 @@ func (r *DeviceReconciler) reconcileUpdateDevice(ctx context.Context, d *devicev
 	}
 
 	// 2. reconciling the device properties' value
-	log.V(3).Info("reconciling the device properties")
+	klog.V(3).Info("reconciling the device properties")
```

Comment:
Adding device name into log message may be better.

Reply:
The deviceName has been added.